### DISCUSSION TITLE
ESLint version ^7.32.0

### DISCUSSION THREAD

**Discussion by KristjanESPERANTO** [super-linter/super-linter#2726] [General]
I am wondering why ESLint is used in a relatively old version (^7.32.0). Is there a specific reason for this?

I ask because I get the following error message: `Environment key "es2022" is unknown`. As far as I understand, this should not occur with the current version.

**Comment by lindluni** [comment]
Released in v4.10.0. We had to contribute a few fixes into ts-standard to support updating our Javascript toolchains.
